docs/source/markdown/podman-run.1.md.in

```diff
@@ -619,7 +619,7 @@ $ podman run --mount type=bind,src=volume-name,target=/data1 busybox sh
 When using SELinux, be aware that the host has no knowledge of container SELinux
 policy. Therefore, in the above example, if SELinux policy is enforced, the
 _/var/db_ directory is not writable to the container. A "Permission Denied"
-message will occur and an **avc:** message in the host's syslog.
+message occurs and an **avc:** message in the host's syslog.
```

Comment:
I think?  Also it probably should be "Syslog", but we have "syslog" all over and then some.

```suggestion
message occurs, and an **avc:** message is added to the host's syslog.
```